README.md

```diff
@@ -121,18 +169,24 @@ To see all rules supported by the makefile, run `make help`
 ```bash
 export NAMESPACE="devworkspace-controller"
 make install
+# Wait for webhook server to start
+kubectl rollout status deployment devworkspace-controller-manager -n $NAMESPACE --timeout 90s
+kubectl rollout status deployment devworkspace-webhook-server -n $NAMESPACE --timeout 90s
+# Scale on-cluster deployment to zero to avoid conflict with locally-running instance
 oc patch deployment/devworkspace-controller-manager --patch "{\"spec\":{\"replicas\":0}}" -n $NAMESPACE
 make run
 ```
 
-When running locally, only a single namespace is watched; as a result, all devworkspaces have to be deployed to `${NAMESPACE}`
-
 ### Run controller locally and debug
 Debugging the controller depends on [delve](https://github.com/go-delve/delve) being installed (`go install github.com/go-delve/delve/cmd/dlv@latest`). Note that `$GOPATH/bin` or `$GOBIN` must be added to `$PATH` in order for `make debug` to run correctly.
 
 ```bash
 make install
+# Wait for webhook server to start
+kubectl rollout status deployment devworkspace-controller-manager -n $NAMESPACE --timeout 90s
+kubectl rollout status deployment devworkspace-webhook-server -n $NAMESPACE --timeout 90s
 oc patch deployment/devworkspace-controller-manager --patch "{\"spec\":{\"replicas\":0}}"
+# Scale on-cluster deployment to zero to avoid conflict with locally-running instance
```

Comment:
Shouldn't this comment appear before `oc patch deployment/devworkspace-controller-manager --patch "{\"spec\":{\"replicas\":0}}"`?